src/DotLiquid/Properties/Resources.resx

```diff
@@ -264,4 +264,7 @@
   <data name="DateFilterRubyDirectiveInvalid" xml:space="preserve">
     <value>Invalid directive passed to {0}</value>
   </data>
+  <data name="ContextPropertyReadonly" xml:space="preserve">
+    <value>Property '{0}' cannot be modified after the context has entered its current state.</value>
```

Comment:
I'm not sure what "entered its current state" means. Is there something more specific we can say here? Does it have explicit states?